_data/sites.json

```
@@ -4654,6 +4665,16 @@
         ]
     },
 
+    {
+        "name": "Mix.tj",
+        "url": "http://mix.tj/info/contact",
+        "difficulty": "impossible",
```

```suggestion
        "difficulty": "hard",
```
According to our guidelines, if you can delete the account after contacting by e-mail, then it is hard, but not impossible.